### PR TITLE
Adding some comments to block module

### PR DESCRIPTION
**Pull request by Phansa**
I noticed a lot of the modules don't have comments/it's hard to figure out what is going on so I thought it would be a good idea to add some comments to help other people trying to develop modules understand what is going on.

### CODE REVIEW

File: modules/blockuser.cpp [hunk 1]
```diff
@@ -178,20 +188,21 @@ class CBlockUser : public CModule {
...
-        // Disconnect all clients
+        //Disconnect all clients
         vector<CClient*> vpClients = pUser->GetAllClients();
         vector<CClient*>::iterator it;
         for (it = vpClients.begin(); it != vpClients.end(); ++it) {
             (*it)->PutStatusNotice(MESSAGE);
             (*it)->Close(Csock::CLT_AFTERWRITE);
         }
 
-        // Disconnect all networks from irc
```
Comment (DarthGandalf) [inline modules/blockuser.cpp]: Why did you remove space?

File: modules/blockuser.cpp [hunk 2]
```diff
@@ -126,6 +133,7 @@ class CBlockUser : public CModule {
             PutModule("This user is not blocked");
     }
 
+    //Compressing all above functions into one for embedded web requests.
```
Comment (DarthGandalf) [inline modules/blockuser.cpp]: It doesn't compress anything...

File: modules/blockuser.cpp [hunk 3]
```diff
@@ -24,6 +24,7 @@ using std::vector;
 class CBlockUser : public CModule {
   public:
     MODCONSTRUCTOR(CBlockUser) {
+        //Creating all commands
```
Comment (DarthGandalf) [inline modules/blockuser.cpp]: I don't see how this comment helps. Name of `AddCommand` function already contains what it does, no?

File: modules/blockuser.cpp [hunk 4]
```diff
@@ -79,6 +82,7 @@ class CBlockUser : public CModule {
         }
     }
 
+    //Displays all blocked users as a list.
```
Comment (DarthGandalf) [inline modules/blockuser.cpp]: There is already "List blocked users" in description of this command, above

File: modules/blockuser.cpp (hunk 3 above)
Comment (Phansa) [inline modules/blockuser.cpp]: Yes I agree/will take it out. Also would this be worthwhile to work on? I think I should have created an issue first, sorry about this.

File: modules/blockuser.cpp (hunk 2 above)
Comment (Phansa) [inline modules/blockuser.cpp]: What i'm trying to say is this that this gives the user the same functionality as the other commands, not sure how to word it or if it is necessary.

**Comment by DarthGandalf** [comment]
Re spaces, there is a clang-format configuration in this repo, so you can run clang-format to automatically fix some things.

File: modules/blockuser.cpp [hunk 5]
```diff
@@ -62,6 +62,8 @@ class CBlockUser : public CModule {
         return true;
     }
 
+    /* If user is the on the blocked list and tries to log in, displays - MESSAGE
```
Comment (DarthGandalf) [inline modules/blockuser.cpp]: the on?

File: modules/blockuser.cpp [hunk 6]
```diff
@@ -168,6 +175,8 @@ class CBlockUser : public CModule {
     }
 
   private:
+    /* Iterates through all blocked users and returns true if they 
+    are blocked, else false*/
```
Comment (DarthGandalf) [inline modules/blockuser.cpp]: This sounds like "if all blocked users are blocked".

File: modules/blockuser.cpp (hunk 5 above)
Comment (Phansa) [inline modules/blockuser.cpp]: Was a typo/I fixed it.
